src/components/projects/projects-table/projects-table.tsx

```diff
@@ -126,11 +112,9 @@ const ProjectsTable: React.FC = () => {
     const leads: User[] = [];
     const seenList: number[] = [];
     for (const project of projects) {
-      for (const user of project.projectLead) {
-        if (!seenList.includes(user.id)) {
-          seenList.push(user.id);
-          leads.push(user);
-        }
+      if (!seenList.includes(project.projectLead!.userId)) {
```

Comment:
This fails when the project lead is missing. Optional chaining is likely needed, but not null assertion can be used at the end to trick the type checker into agreeing that things make sense.